### ISSUE TITLE
Is there any way to set text field or other fields as read only?

### ISSUE THREAD

**Issue by rahulmahadik** [danvick/flutter_form_builder#16]
wants to create read only text field

```
FormBuilderInput.textField(
      type: FormBuilderInput.TYPE_TEXT,
      attribute: "name",
      label: "Name",
      require: true,
      min: 3,
    ),
```
Now i am using `validator` option. where i check the original value of text field while submit form.

**Comment by danvick** [comment]
This is a feature planned for future releases. Thank you

**Comment by danvick** [comment]
Closed by release v1.4.0
